katello_certs_tools/katello_ssl_tool.py

```diff
@@ -906,19 +947,21 @@ def genServerRpm(d, verbosity=0):
         _reenableRpmMacros()
         os.unlink(postun_scriptlet)
 
-    out = out_stream.read(); out_stream.close()
-    err = err_stream.read(); err_stream.close()
+    out = out_stream.read()
+    out_stream.close()
+    err = err_stream.read()
+    err_stream.close()
 
     if ret or not os.path.exists("%s.noarch.rpm" % serverRpmName):
         raise GenServerRpmException("web server's SSL key set RPM generation "
                                     "failed:\n%s\n%s" % (out, err))
     if verbosity > 2:
         if out:
-            print "STDOUT:", out
+            print("STDOUT:", out)
         if err:
-            print "STDERR:", err
+            print("STDERR:", err)
 
-    os.chmod('%s.noarch.rpm' % serverRpmName, 0600)
+    os.chmod('%s.noarch.rpm' % serverRpmName, '0600')
```

Comment:
`os.chmod` does not accept string parameters (at least not in Python 3.6 and 3.7)